db/migrate/20150316174702_create_reviews.rb

```diff
@@ -0,0 +1,15 @@
+class CreateReviews < ActiveRecord::Migration
+  def change
+    create_table :reviews do |t|
+      t.string :priority_id
```

Comment:
I don't think this is actually needed.. We don't have any priority_id for a review